tailwind.config.js

```diff
@@ -1,8 +1,12 @@
 const colors = require("tailwindcss/colors");
 
 module.exports = {
-  mode: "jit",
-  purge: ["./app/**/*.html.erb", "./app/**/*.bs.js", "./app/**/*.jsx"],
+  content: ["./app/**/*.html.erb", "./app/**/*.bs.js", "./app/**/*.jsx"],
+  safeList: [
```

Comment:
This key is incorrect. [According to Tailwind's documentation](https://tailwindcss.com/docs/content-configuration#safelisting-classes), the correct key is `safelist` (small letter `l`).

The use of a safe-list is something we should look to remove anyway, since this goes against the Tailwind's recommendations. If this isn't required now, then we can go ahead and remove this.

Reply:
The safelist is required for now. We have many instances in the code where we are composing class names dynamically.

Reply:
@irajsuhail The key is still incorrect, though..? 😕

Reply:
Yep, The key is incorrect.